lib/websock.ex

```diff
@@ -81,8 +78,10 @@ defmodule WebSock do
   * `{:stop, reason :: term(), close_detail(), state()}`: Similar to the previous clause, but allows
     for the explicit setting of either a plain close code or a close code with a body to be sent to
     the client
+  * `{:stop, reason :: term(), close_detail(), state()}`: Similar to the previous clause, but allows
```

Comment:
Just noticed that this doc item references incorrect spec part (4-tuple, not the 5-tuple with messages).

```suggestion
  * `{:stop, reason :: term(), close_detail(), messages(), state()}`: Similar to the previous clause, but allows
```

Reply:
yeesh. I'm just full of 'em today.

Reply:
Fixed.

Reply:
I've reread the PR and I think it looks great, no further comments :)